### DISCUSSION TITLE
Revert an Inherited Field to "Inheriting"

### DISCUSSION THREAD

**Discussion by cancan101** [pimcore/pimcore#16943] [Q&A / Support] [answered]
Once an inherited fields has been edited, is there anyway to clear out the local changed and go back to "inheriting from parent"?

**Comment by fashxp** [comment, marked as answer]
empty the field

**Reply by cancan101** [reply]
That worked.  I did have to reload the Object page after clearing the field and saving to see that inheritance was happening.
